plugins/chunter-resources/src/components/PinnedMessagesPopup.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+  import { ChunterMessage } from '@anticrm/chunter'
+  import contact, { Employee, EmployeeAccount, formatName } from '@anticrm/contact'
+  import { Ref } from '@anticrm/core'
+  import { Avatar, getClient, MessageViewer } from '@anticrm/presentation'
+  import { IconClose } from '@anticrm/ui'
+  import { createEventDispatcher } from 'svelte'
+  import { getTime } from '../utils'
+
+  export let pinnedMessages: ChunterMessage[]
+  export let employeeAcounts: EmployeeAccount[]
+
+  const client = getClient()
+  const dispatch = createEventDispatcher()
+
+  function getAvatar (id?: Ref<Employee>): string | null {
+    let avatar: string | null = null
+    client.findOne<Employee>(contact.class.Employee, { _id: id }).then((e) => (avatar = e?.avatar ?? null))
+    return avatar
+  }
+
+  function getEmployee (message: ChunterMessage): EmployeeAccount | undefined {
+    return employeeAcounts?.find((e) => e._id === message.createBy)
+  }
+</script>
+
+<div class="vScroll popup">
+  {#each pinnedMessages as message}
+    <div class="message">
+      <div class="header">
+        {#await getEmployee(message) then employeeAccount}
+          {#await getAvatar(employeeAccount?.employee) then avatar}
+            <div class="avatar">
+              <Avatar size={'medium'} {avatar} />
+            </div>
+          {/await}
+          <span class="name">
+            {formatName(employeeAccount?.name ?? '')}
+          </span>
+        {/await}
+        <div
+          class="cross"
+          on:click={() => {
+            dispatch('close', { message: message })
+          }}
+        >
+          <IconClose size="small" />
+        </div>
+      </div>
+      <MessageViewer message={message.content} />
+      <span class="time">{getTime(message.createOn)}</span>
+    </div>
+  {/each}
+</div>
+
+<style lang="scss">
+  .popup {
+    display: flex;
+    flex-direction: column;
+    padding: 1.25rem 1.25rem 1.25rem;
+    width: 300px;
```

Comment:
Please don't use `px` and `rem` in one place. Try use styles from `packages/theme/styles`